Limelight/ViewControllers/StreamFrameViewController.m

```diff
@@ -61,8 +90,9 @@ - (void)viewDidLoad
     
     _controllerSupport = [[ControllerSupport alloc] initWithConfig:self.streamConfig presenceDelegate:self];
     _inactivityTimer = nil;
-    
+    _renderView = (StreamView*)[[UIView alloc] initWithFrame:self.view.frame];
```

Comment:
It's not safe to cast a UIView to a StreamView.

Reply:
What would you suggest to be a better alternative? Should it be a split to a `#IF os(iOS)`? Is this due to StreamView being a possible OSView for non iOS devices?